2023-01-01T13:49:50.4464106+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2023-01-01T13:49:50.4771339+05:30  [INF] Hosting environment: "Development" (c3307c92)
2023-01-01T13:49:50.4803311+05:30  [INF] Content root path: "C:\MyProject\Vehicle-mishap\ODPortalWebAPI" (b5d60022)
2023-01-01T13:49:50.6355474+05:30 8000003f-0003-fb00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2023-01-01T13:50:12.7041045+05:30 80000149-0004-f900-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
ODPortalWebDL.DTO.ExceptionModal.CustomException: Invalid Login details entered.
   at ODPortalWebDL.DataAccess.UserManagerDataAccess.CheckUserPassWord(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\DataAccess\UserManagerDataAccess.cs:line 63
   at ODPortalWebDL.Manager.UserManager.Auth(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\Manager\UserManager.cs:line 26
   at ODPortalWebAPI.Controllers.UserController.Authenticate(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ODPortalWebAPI.Startup.<>c.<<Configure>b__5_1>d.MoveNext() in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Startup.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2023-01-01T13:51:39.9258204+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2023-01-01T13:51:39.9588783+05:30  [INF] Hosting environment: "Development" (c3307c92)
2023-01-01T13:51:39.9739264+05:30  [INF] Content root path: "C:\MyProject\Vehicle-mishap\ODPortalWebAPI" (b5d60022)
2023-01-01T13:51:40.0386281+05:30 800000f1-0000-fe00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2023-01-01T13:52:01.1713629+05:30 80000065-0006-fc00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
ODPortalWebDL.DTO.ExceptionModal.CustomException: Invalid Login details entered.
   at ODPortalWebDL.DataAccess.UserManagerDataAccess.CheckUserPassWord(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\DataAccess\UserManagerDataAccess.cs:line 63
   at ODPortalWebDL.Manager.UserManager.Auth(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\Manager\UserManager.cs:line 26
   at ODPortalWebAPI.Controllers.UserController.Authenticate(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ODPortalWebAPI.Startup.<>c.<<Configure>b__5_1>d.MoveNext() in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Startup.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2023-01-01T13:54:01.2687992+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2023-01-01T13:54:01.3036143+05:30  [INF] Hosting environment: "Development" (c3307c92)
2023-01-01T13:54:01.3067335+05:30  [INF] Content root path: "C:\MyProject\Vehicle-mishap\ODPortalWebAPI" (b5d60022)
2023-01-01T13:54:01.4262621+05:30 80000013-0002-fb00-b63f-84710c7967bb [WRN] Failed to determine the https port for redirect. (ca76cc21)
2023-01-01T13:54:56.4058778+05:30 80000046-0003-fb00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
ODPortalWebDL.DTO.ExceptionModal.CustomException: Invalid Login details entered.
   at ODPortalWebDL.DataAccess.UserManagerDataAccess.CheckUserPassWord(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\DataAccess\UserManagerDataAccess.cs:line 63
   at ODPortalWebDL.Manager.UserManager.Auth(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebDL\Manager\UserManager.cs:line 26
   at ODPortalWebAPI.Controllers.UserController.Authenticate(Credentials credentials) in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Controllers\UserController.cs:line 30
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at ODPortalWebAPI.Startup.<>c.<<Configure>b__5_1>d.MoveNext() in C:\MyProject\Vehicle-mishap\ODPortalWebAPI\Startup.cs:line 59
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
